src/app/components/Home/GroupSelects/components/index.ts

```diff
@@ -0,0 +1,6 @@
+import InputItem from "./InputItem";
+import LabelForInput from "./LabelForInput";
+import Select from "./Select";
+import WrapperForSelect from "./WrapperForSelect";
+
```

Comment:
Can be done this way:
`exprot { default as InputItem } from "./InputItem";` ...